proxy/Dockerfile

```
@@ -1,5 +1,5 @@
-# syntax=docker/dockerfile:1.5
-FROM nginxinc/nginx-unprivileged:1.27.5-alpine
+# syntax=docker/dockerfile:1.15
+FROM nginxinc/nginx-unprivileged:1.29.0-alpine
 
 USER root
```

Hadolint (2.12.0)

[warning] Last USER should not be root

(DL3002)